components/uniset/src/uniset.rs

```diff
@@ -192,18 +201,19 @@ impl UnicodeSet {
 
 #[cfg(test)]
 mod tests {
-    use super::{UnicodeSet, BMP_MAX};
+    use super::{USetError, UnicodeSet, BMP_MAX};
     use std::{char::MAX, convert::TryFrom, vec::Vec};
 
     #[test]
     fn test_unicodeset_try_from_vec() {
-        let check = vec![2, 3, 4, 5];
-        assert!(UnicodeSet::try_from(check).is_ok());
+        let check = UnicodeSet::try_from(vec![2, 3, 4, 5]).unwrap().inv_list;
+        assert_eq!(vec![2, 3, 4, 5], check);
     }
     #[test]
     fn test_unicodeset_try_from_vec_error() {
         let check = vec![1, 1, 2, 3, 4];
-        assert!(UnicodeSet::try_from(check).is_err());
+        let set = UnicodeSet::try_from(vec![1, 1, 2, 3, 4]);
```

Comment:
Here and elsewhere: you can use `.clone()`:

`let set = UnicodeSet::try_from(check.clone());`

Reply:
agreed and changed